scripts/configuration/config-parser.cake

```diff
@@ -6,6 +6,8 @@ using System.Collections.Generic;
 public static class ConfigFile
 {
     public const string Path = "scripts/configuration/ac-build-config.xml";
```

Comment:
Reuse `Name ` here

Reply:
It should be a const. If I use a variable it isn't and fails build.